clang/lib/Sema/SemaSYCL.cpp

```diff
@@ -2871,6 +2970,14 @@ class SyclKernelIntHeaderCreator : public SyclKernelFieldHandler {
     return true;
   }
 
+  void handleSyclKernelHandlerType(QualType Ty) {
+    // Add corresponding entry in integration header.
```

Comment:
Hmm.... I guess the offset is simply nonsensical in this case, right?  I wonder if we're better off having some sort of sentinel value to mean that.

Reply:
I think the runtime team will need to answer that. I do not really know how the integration header is consumed

Reply:
I know the runtime just loops through these values, but it seems that the offset is simply meaningless/won't be used.  From a practicality perspective, pushing `0` here is likely fine.  That said (assuming I'm right here), I'd vastly prefer it be an obviously 'meaningless' number.  I think that it is unsigned, so -1 doesn't work, so unsigned-max is likely our choice.  Please talk with @romanovvlad and confirm I'm correct in its meaninglessness.

Reply:
I haven't made this change. I thought I would complete a round of testing with 0 and then change it to unsigned-max to see if change causes any break. @romanovvlad mentioned he thinks offset is meaningless based on comment here - https://github.com/intel/llvm/pull/3331/files#r593198110. If that is the case, I think spec should be modified before we change the implementation here